### PR TITLE
Collapsible section

### PR DESCRIPTION
**Pull request by imee12**
collapsible sections
1: all can be open
2. only one can be open

### CODE REVIEW

**Comment by igreulich** [comment]
You didn't include jquery in the html file, I can't test it.

**Comment by imee12** [comment]
<script src="node_modules/jquery/dist/jquery.js"></script>
 this is what i included in collapsible.html line 4.. ^^ should i include it differently?

**Comment by igreulich** [comment]
For this, you can just grab a cdn link from somewhere, no need to bring in node_modules for this.

--
Ian Greulich

File: collapsible_section/collapsible.less [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+.collapsible-section {
+  font-family: @font-family-normal;
+
+  & > label {
+    padding: .7rem 1rem;
+    background-color: @blue;
+    color: white;
+    font-weight: bold;
+    font-family: 'Merriweather-Bold';
```
Comment (igreulich) [inline collapsible_section/collapsible.less]: we should probably variableize this when it gets pulled into actual js code.

File: collapsible_section/collapsible.less [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    display: inline;
+  }
+  & ~ label .fa-plus-square {
+    display: none;
+  }
+
+}
+
+  .collapsible-section-handler {
+    position: absolute;
```
Comment (igreulich) [inline collapsible_section/collapsible.less]: I feel like this is a thing we have done enough we should consider making it a helper.

**Pull request merged**